sys/include/entropy_source/entropy_source.h

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup   sys_entropy_source_common Entropy Source Common
+ * @ingroup    sys_entropy_source
+ * @brief      Common definitions and functions for entropy sources
+ *
+ * @{
+ * @file
+ *
+ * @author      Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ */
+
+#ifndef ENTROPY_SOURCE_H
+#define ENTROPY_SOURCE_H
```

Comment:
```suggestion
#ifndef ENTROPY_SOURCE_ENTROPY_SOURCE_H
#define ENTROPY_SOURCE_ENTROPY_SOURCE_H
```

But I would rather suggest to move the file to `sys/include/entropy_source.h`